test/WebSites/BasicWebSite/Controllers/RemoteAttribute_HomeController.cs

```diff
@@ -1,16 +1,14 @@
-// Copyright (c) .NET Foundation. All rights reserved.
+﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using BasicWebSite.Models;
 using Microsoft.AspNet.Mvc;
-using ValidationWebSite.Models;
 
-namespace ValidationWebSite.MyArea.Controllers
+namespace BasicWebSite.Controllers
 {
-    [Area("Aria")]
-    [Route("[Area]/[Controller]/[Action]", Order = -2)]
```

Comment:
Why remove attribute routing here? This code illustrated a workaround for an interesting routing interaction.